google-cloud-firestore/lib/google/cloud/firestore.rb

```diff
@@ -21,6 +21,442 @@ module Cloud
     ##
     # # Cloud Firestore
     #
+    # Cloud Firestore is a NoSQL document database built for automatic scaling,
+    # high performance, and ease of application development. While the Cloud
+    # Firestore interface has many of the same features as traditional
+    # databases, as a NoSQL database it differs from them in the way it
+    # describes relationships between data objects.
+    #
+    # For more information about Cloud Firestore, read the [Cloud
+    # Firestore Documentation](https://cloud.google.com/firestore/docs/).
+    #
+    # The goal of google-cloud is to provide an API that is comfortable to
+    # Rubyists. Authentication is handled by {Google::Cloud#firestore}. You can
+    # provide the project and credential information to connect to the Cloud
+    # Firestore service, or if you are running on Google Compute Engine this
+    # configuration is taken care of for you. You can read more about the
+    # options for connecting in the [Authentication
+    # Guide](https://googlecloudplatform.github.io/google-cloud-ruby/#/docs/guides/authentication).
+    #
+    # ## Adding data
+    #
+    # Cloud Firestore stores data in Documents, which are stored in Collections.
+    # Cloud Firestore creates collections and documents implicitly the first
+    # time you add data to the document. You do not need to explicitly create
+    # collections or documents. (For more information, see [Adding Data to Cloud
```

Comment:
Nit: I would remove "You do not need to explicitly create collections or documents." since it's stated right before.

Reply:
Removed.

Reply:
Fixed.